internal/hud/server/server.go

```diff
@@ -64,11 +68,19 @@ func ProvideHeadsUpServer(
 	analytics *tiltanalytics.TiltAnalytics,
 	uploader cloud.SnapshotUploader) *HeadsUpServer {
 	r := mux.NewRouter().UseEncodedPath()
+	grpcMux := runtime.NewServeMux()
 	s := &HeadsUpServer{
 		store:    store,
 		router:   r,
 		a:        analytics,
 		uploader: uploader,
+		grpcMux:  grpcMux,
+	}
+
+	err := proto_webview.RegisterViewServiceHandlerServer(context.TODO(), grpcMux, s)
+	if err != nil {
+		// TODO(dmiller) what should we do here?
```

Comment:
let's change this provider to return an error and re-run `wire`. it should handle this correctly.